Symfony/src/AppBundle/Entity/User.php

```diff
@@ -32,6 +38,11 @@ class User implements UserInterface, \Serializable
      * @var string
      *
      * @ORM\Column(name="email", type="string", length=255, unique=true)
+     *
+     * @Assert\Length(
```

Comment:
Got it, added backend validation and improved the error display on both login and register forms :)